src/main/drivers/display_ug2864hsweg01.c

```diff
@@ -274,6 +293,43 @@ void i2c_OLED_send_string(busDevice_t *bus, const char *string)
     }
 }
 
+void i2c_OLED_send_string_vertical(busDevice_t *bus, uint8_t x, uint8_t y, const char *string)
+{
+    // vertical mode
+    i2c_OLED_set_vertical_adressing_mode(bus);
+
+    uint8_t buffer[SCREEN_HEIGHT/8];
+
+    // Sends a string of chars until null terminator
+    const char *sptr;
+    uint8_t i;
```

Comment:
Use `int` or `unsigned` it will generate code that is shorted and faster. ARM is not Arduino.

It's also better to define variables in smallest scope (for loop in this case)